### PR TITLE
[Fix] Notifications settings inverted

### LINKED ISSUE #10329: 🐛 Notifications settings UI is backwards
## 🐛 Bug

On the new Account page the Notifications Settings section is showing the incorrect values. The checkbox values are backwards, so it's showing true for the boxes we want to ignore. Shout out to @petertgiles for catching this 🔎.

## 📸 Screenshot

image (8)

## ✅ Acceptance Criteria

- [ ] Ensure the notification default values are false if the user wants to ignore the notification type, and vise versa.

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #10329 

## 👋 Introduction

Updates the notification settings to display the proper value in the check boxes.

## 🕵️ Details

This also fixes the link from the notifications page to notification settings.

## 🧪 Testing

1. Build `pnpm run dev`
2. Go to your notifications page
3. Click on the "Update settings" link in the sidebar
4. Confirm it takes you to the Notification settings section on the Account settings page
5. Update settings
6. Confirm the check boxes are checked for items not in the ignored array

### CODE REVIEW

File: apps/web/src/pages/Profile/AccountSettings/utils.ts [hunk 1]
```diff
@@ -1,108 +1,79 @@
...
+    systemMessages: ["email", "inApp"],
+    applicationUpdates: getIgnoredNotificationFamilyValue(
+      ignoredNotifications,
+      NotificationFamily.ApplicationUpdate,
+    ),
+    jobAlerts: getIgnoredNotificationFamilyValue(
+      ignoredNotifications,
+      NotificationFamily.JobAlert,
+    ),
+  } satisfies FormValues;
```
Comment (petertgiles) [inline apps/web/src/pages/Profile/AccountSettings/utils.ts:78]: Ooh, `satisfies`, nice.

File: apps/web/src/pages/Notifications/NotificationsPage/NotificationsPage.tsx [hunk 2]
```diff
@@ -67,9 +67,8 @@ const NotificationsPage = () => {
                   })}
                 </p>
                 <p>
-                  {/** Update URL when page is created in (ticket not created yet?) */}
                   <Link
-                    href={paths.home()}
```
Comment (petertgiles) [inline apps/web/src/pages/Notifications/NotificationsPage/NotificationsPage.tsx:72]: Thanks for tacking this on.

**Review by petertgiles — APPROVED** [review]
Thanks for fixing this!

**Review by yonikid15 — APPROVED** [review]
Nice 👍

**Pull request merged**